src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -1610,17 +1614,38 @@ void AllocateSurfaceHeatBalArrays(EnergyPlusData &state)
                                 OutputProcessor::SOVTimeStepType::Zone,
                                 OutputProcessor::SOVStoreType::State,
                                 surface.Name);
+            SetupOutputVariable(state,
+                                "Surface Inside Face Lights Radiation Heat Gain Rate From Other Zones",
+                                OutputProcessor::Unit::W,
+                                state.dataHeatBalSurf->SurfQdotRadLightsInRepOtherZones(loop),
+                                OutputProcessor::SOVTimeStepType::Zone,
+                                OutputProcessor::SOVStoreType::State,
+                                surface.Name);
             SetupOutputVariable(state,
                                 "Surface Inside Face Lights Radiation Heat Gain Rate per Area",
                                 OutputProcessor::Unit::W_m2,
                                 state.dataHeatBalSurf->SurfQdotRadLightsInPerArea(loop),
                                 OutputProcessor::SOVTimeStepType::Zone,
                                 OutputProcessor::SOVStoreType::State,
                                 surface.Name);
+            SetupOutputVariable(state,
+                                "Surface Inside Face Lights Radiation Heat Gain Rate per Area from Other Zones",
```

Comment:
I'm not sure this is the best solution. We don't have a separate output variable for Solar Gain from Other Zones, do we?
The better solution which may or may not be feasible would be to track lights visible energy through interzone windows separately from solar. That way the unexpected lighting energy would move from "Surface Inside Face Solar Radiation Heat Gain" to "Surface Inside Face Lights Radiation Heat Gain".

Reply:
I guess I had not considered that solution when I worked on this a while ago.  I didn't really understand why it was done that way to be honest.  It seemed like it would set things up for a potential problem...which it did.  I have to admit that it's been a while since I worked on this so I'd have to dive back in here and re-figure things out.  If that's what's needed, then I'll take a stab at it.  Thoughts?